src/sql/workbench/api/node/mainThreadDataProtocol.ts

```diff
@@ -278,6 +278,16 @@ export class MainThreadDataProtocol implements MainThreadDataProtocolShape {
 		return undefined;
 	}
 
+	public $registerIconProvider(providerId: string, handle: number): Promise<any> {
+		const self = this;
+		this._connectionManagementService.registerIconProvider(providerId, <azdata.IconProvider>{
```

Comment:
nitpick: space between `<azdata.IconProvider> {` please

Reply:
The space between them causes commit failure.
```
> husky - npm run -s precommit
> husky - node v8.15.0

Reading git index versions...
File not formatted. Run the 'Format Document' command to fix it: src\sql\workbench\api\node\mainThreadDataProtocol.ts


Hygiene failed with 1 errors. Check 'build/gulpfile.hygiene.js'.

> husky - pre-commit hook failed (add --no-verify to bypass)
> husky - to debug, use 'npm run precommit'
```